romancal/source_catalog/tests/test_source_catalog.py

```diff
@@ -324,8 +324,8 @@ def test_l3_input_model_unchanged(mosaic_model, tmp_path):
         save_results=False,
     )
 
-    assert_allclose(original_data, mosaic_model.data, atol=5.0e-5)
-    assert_allclose(original_err, mosaic_model.err, atol=5.0e-5)
+    assert_equal(original_data, mosaic_model.data)
```

Comment:
I think it would be safer to use `assert_allclose()` because both arrays are of type `float32`.

Reply:
same as above